### DISCUSSION TITLE
Should I install this if I just want Tidal connect in the Tidal app?

### DISCUSSION THREAD

**Discussion by Hessam-HK** [EbbLabs/mopidy-tidal#165] [Q&A] [answered]
Hi,

So, yeah... the question is really simpel. If i just want my Raspberry Pi 2 with piCorePlayer 8.2.0 to show in the Tidal app under Tidal connect, should install this app on the Pi, or another app?

Bare in mind, I'm a total noob to Pi's, Linux and piCorePlayer.

Thanks. 🙂

**Comment by tehkillerbee** [comment, marked as answer]
Hello,

No problem! No this project won't work as it is not meant to provide tidal-connect functionality. Instead it will give you a way to control tidal playback using a web frontend such as Mopidy-Iris. 

For instance, you could have a (headless) RPi connected to your home stereo and use mopidy to control playback.

If you want a tidal-connect solution, you could take a look here: https://github.com/GioF71/tidal-connect or perhaps https://github.com/TonyTromp/tidal-connect-docker

**Reply by Hessam-HK** [reply]
Thank you for the quick answer, tehkillerbee. So what you're saying is, if I had a RPi with a tidal app installed, hooked up to my home stereo I could control Tidal via Mopidy? 😊

Funny... one the repositories you are referring to, is referring back to this repository as an alternative. 😁

**Reply by tehkillerbee** [reply]
Yes, if you install mopidy-tidal on your RPi, then you can navigate your Tidal library, control playback etc through mopidy. When paired with a web frontend, you can use your phone or any other device that can access your RPi to access it. Not all functionality is available but with the mopidy-Iris frontend installed, the user experience is pretty decent. 

Correct, I linked to both as I have no experience using either but I believe the first link should provide you with a more user friendly setup experience. :)
